README.md

```diff
@@ -106,6 +106,7 @@ playground for learning and exploring.
 * **[MD048](doc/Rules.md#md048)** *code-fence-style* - Code fence style
 * **[MD049](doc/Rules.md#md049)** *emphasis-style* - Emphasis style should be consistent
 * **[MD050](doc/Rules.md#md050)** *strong-style* - Strong style should be consistent
+* **[MD051](doc/Rules.md#md051)** *no-dead-hash-links-within-document* - No dead hash links within document
```

Comment:
How about just calling the rule `valid-link-fragments`?
https://spec.commonmark.org/0.30/#example-500

Reply:
"Link fragments should be valid"